.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+<!--
+This form is for bug reports and feature requests ONLY!
```

Comment:
I think I'd get rid of this line; it's a bit unfriendly and carries the same meaning as the next line about contacting AWS Support.